modin/backends/pandas/query_compiler.py

```diff
@@ -123,14 +123,16 @@ def dt_op_builder(df, *args, **kwargs):
     return dt_op_builder
 
 
-def copy_df_for_func(func):
+def copy_df_for_func(func, func_name=None):
```

Comment:
```suggestion
def copy_df_for_func(func, func_name:str=None):
```
with whatever formatting `black` wants

Reply:
Okay, will be fixed.